### PR TITLE
gst-va:Add 8/10bit VVC decode test

### PR DESCRIPTION
**Pull request by Hangjie22Coder**

### CODE REVIEW

File: lib/codecs.py [hunk 1]
```diff
@@ -19,6 +19,7 @@ class Codec(str, enum.Enum):
   MJPEG = "mjpeg"
   MPEG2 = "mpeg2"
   VC1   = "vc1"
-
+  VVC_8   = "vvc_8"
+  VVC_10  = "vvc_10"
```
Comment (uartie) [inline lib/codecs.py:23]: This should not include the bitdepth indicator.
```suggestion
  VVC   = "vvc"
```

File: lib/gstreamer/va/decoder.py [hunk 2]
```diff
@@ -36,6 +36,8 @@ def decode_test_class(codec, bitdepth, **kwargs):
   gstcodec = {
     Codec.AVC   : "h264",
     Codec.HEVC  : "h265",
+    Codec.VVC_8   : "h266",
+    Codec.VVC_10   : "h266",
```
Comment (uartie) [inline lib/gstreamer/va/decoder.py:40]:
```suggestion
    Codec.VVC   : "h266",
```

File: lib/gstreamer/va/decoder.py [hunk 3]
```diff
@@ -90,3 +92,7 @@ def validate_caps(self):
 
 ## MPEG2 ##
 MPEG2DecoderTest    = decode_test_class(codec = Codec.MPEG2, bitdepth = 8)
+
+## VVC ##
+VVC_8DecoderTest    = decode_test_class(codec = Codec.VVC_8, bitdepth = 8)
+VVC_10DecoderTest   = decode_test_class(codec = Codec.VVC_10, bitdepth = 10)
```
Comment (uartie) [inline lib/gstreamer/va/decoder.py:98]:
```suggestion
## VVC ##
VVC_8DecoderTest    = decode_test_class(codec = Codec.VVC, bitdepth = 8)
VVC_10DecoderTest   = decode_test_class(codec = Codec.VVC, bitdepth = 10)
```

**Review by uartie — CHANGES_REQUESTED** [review]

File: lib/gstreamer/va/decoder.py (hunk 2 above)
Comment (uartie) [reply lib/gstreamer/va/decoder.py:40]: Also, we need to add `Codec.VVC` to the `capcodec` condition (i.e. "caps lookup translation")... located at the top of this function.

File: lib/codecs.py (hunk 1 above)
Comment (uartie) [reply lib/codecs.py:23]: You need to add `Codec.VVC` to the `capcodec` condition (i.e. "caps lookup translation")... located at the top of the decode_test_class function in lib/gstreamer/va/decoder.py

Comment (Hangjie22Coder) [reply lib/codecs.py]: Thank you for the review, it has been updated.

**Review by uartie — APPROVED** [review]

**Pull request merged**